### DISCUSSION TITLE
How to maintain products from Datafeedr and Amazon in the same WooCommerce category

### DISCUSSION THREAD

**Discussion by e-orlov** [datafeedr/wordpress-plugins#28] [Help] [answered]
Hello!

In short I will have a challenge to import, update and maintain in the same WooCommerce category products, which will come from two sources: Datafeedr and Amazon. My current plan is to manage Amazon product in WooCommerce with help of [this plugin](https://www.wzone-plugin.com/).

Does somebody already got such challenge working? Does anybody have recommendations for me? 

I desperately look for a workflow to get this done, without to run in issues like messed updates, overwritten products and so on... Currently I'm free in the selection of Amazon managing plugin and open to all advices.

I realized the Datafeedr API has a possibility to maintainan Amazon product search - does somebody applied it to achieve the goal I describe?

**Comment by EricBusch** [comment, marked as answer]
Hi

Yes, we have quite a few members with this set up and no problems. Just use our plugins to import your non-Amazon products into your WooCommerce store and use Wzone to import the Amazon products. 

No conflicts as far as I know.

If you have any other questions, please don't hesitate to ask.

Thanks,
Eric

**Reply by e-orlov** [reply]
Thank you, Eric, for inspiring answer and outstanding support!

Do you probably know, what else use other users to maintain Amazon products, beside Wzone, and have an opinion or recommendation for me?

**Reply by EricBusch** [reply]
I am only aware of the Wzone plugin. I don't know of any others... yet.

**Reply by e-orlov** [reply]
That is highly important insight ! Thank you again 👍
